docs/metrics.md

```diff
@@ -0,0 +1,83 @@
+# Calculated Metrics
+
+These metrics are calculated using stricly `changes`, `deployments` and `incidents` for a rolling window of 90 days.
+
+## Deployment Frequency
+
+Median of the most specific that applies:
+
+| Deployment Frequency | Value                                                    |
+| -------------------- | -------------------------------------------------------- |
+| Yearly               | ???                                                      |
+| Monthly              | One or more calendar week with `deployments`             |
+| Weekly               | One or more `deployments` in a calendar week             |
+| Daily                | Three or more days with `deployments` in a calendar week |
+| On-demand            | ???                                                      |
```

Comment:
I would put the table in the opposite order since this is like an if else.
It will try to match the first and then if not try the next one.